src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -1112,6 +1093,7 @@ public int hashCode() {
      * @return a boolean indicating whether the collection has changed or not.
      * @throws NullPointerException if the collection or iterator is null
      */
+    @SuppressWarnings("unchecked")
```

Comment:
I might still be missing something here... Reviewing online, it didn't seem to be necessary. Just imported the code to Eclipse, and the project has 41 Warnings. This annotation being one of the warnings, being marked as "Unnecessary @SuppressWarnings("unchecked")".